### ISSUE TITLE
Should we finally remove tutorials from this repo?

### ISSUE THREAD

**Issue by santisoler** [simpeg/simpeg#1555]
Since we started moving the simpeg tutorials to https://github.com/simpeg/user-tutorials, might be a good idea to consider finally removing them from this repository.

By removing the tutorials from this repo we wouldn't need to duplicate efforts maintaining them in two different places. Moreover, building the docs for simpeg will be significantly faster since we won't need to rerun the tutorials.

On the other hand, we would need to be more thorough on updating the user tutorials in sync with simpeg releases.

We should also consider that tutorials are also a place to catch possible bugs. By removing them from this repo we would delay the discovery of those bugs to the next time we update the user-tutorials repo. Nonetheless, I think this is ok, tests should be in charge of catching bugs, and tutorials should be limited to educational resources. If we catch a bug on a tutorial, that's good, but it's not their final goal.

In summary, **should we start removing tutorials from this repo?**

**Comment by jcapriot** [comment]
Yes I agree.

Regarding using them to identify bugs, I think when we remove them, we should be aware of the newly missing lines in the coverage report and take steps to then re-cover those lines with new tests.

**Comment by santisoler** [comment]
As discussed with @lheagy and @dccowan, we could replace the tutorial pages with pages that redirects users to the User Tutorials. We were also talking about removing tutorials by pieces instead of removing them all together, making sure that we have similar content in User Tutorials.
